tests/data/rmg_simulator_test/input.yml

```diff
@@ -0,0 +1,47 @@
+project: T3_minimal
```

Comment:
Please change the commit message to `Tests: Added RMG Simulator Adapter tests` (see https://github.com/ReactionMechanismGenerator/T3/wiki/Contributor-Guidelines#commits-style)